src/proto/mgmt/svc.proto

```diff
@@ -68,14 +68,41 @@ message JoinResp {
 }
 
 message LeaderQueryReq {
-	string system = 1;		// System name.
+	string system = 1;	// System name.
 }
 
 message LeaderQueryResp {
 	string currentLeader = 1;
 	repeated string replicas = 2;
 }
 
+// RASEvent describes a RAS event in the DAOS system.
+message RASEvent {
+	string name = 1;		// Unique identifier, 64-char.
+	string timestamp = 2;		// Fully qualified timestamp (us) incl timezone.
+	uint32 severity = 3;		// Event severity Fatal/Warning/Error/Info.
+	string msg = 4;			// Human readable message describing event.
+	uint32 rank = 5; 		// (optional) DAOS rank involved in event.
+	string hostname = 6;		// (optional) Hostname of node involved in event.
+	bytes data = 7;			// (optional) Instance data treated as blob.
+	uint32 id = 8;			// Unique numeric event identifier.
+	uint32 type = 9;		// Event type StateChange/InfoOnly.
+	uint32 instance_idx = 10;	// (optional) DAOS harness instance index.
```

Comment:
It might be nicer to order these with all of the optional fields at the end. I also recommend against making specific examples of the field usage here, as they are likely to get bit-rotten as the usage expands and evolves.

Reply:
done